backend/services/gemini_ai.py

```
@@ -0,0 +1,146 @@
+import logging
+import os
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)